docker-compose.evm.yml

```diff
@@ -26,6 +30,8 @@ services:
         "echo 'Deleting old sidecars files.';find /sidecar-files/ -type f -delete;echo 'This container is intentionally disabled by the EVM profile.';"
       ]
     command: []
+    network_mode: none
```

Comment:
When starting local node I'm getting this error
```
service record-sidecar-uploader declares mutually exclusive `network_mode` and `networks`: invalid compose project
```

Reply:
Strange, no network option was present for this service. However, I've changed it. Could you please check it once again? @isavov